src/mon/MDSMonitor.h

```diff
@@ -73,7 +73,7 @@ class MDSMonitor : public PaxosService {
   FSMap pending_fsmap;  // current + pending updates
 
   // my helpers
-  template<int dblV = 7>
+  template<int dblV>
```

Comment:
What about killing [the redefinition in MDSMonitor.cc](https://github.com/wjwithagen/ceph/blob/c6ecb1a0e51d62ca7b9899038b5d36ddd4cd053e/src/mon/MDSMonitor.cc#L79) instead?